docs/htdocs/info/docs/tools/vep/script/vep_cache.html

```diff
@@ -112,12 +112,6 @@ <h3 id="manual_cache">Manually downloading caches</h3>
         <p>Essential for human and other species with large sets of variant data - requires <a href="https://github.com/Ensembl/Bio-DB-HTS" rel="external">Bio::DB::HTS</a> (setup by INSTALL.pl) or <a href="https://github.com/samtools/tabix" rel="external">tabix</a>, e.g.:</p>
       <pre class="code sh_sh left-margin">cd $HOME/.vep
 curl -O [[SPECIESDEFS::ENSEMBL_FTP_URL]]/release-[[SPECIESDEFS::ENSEMBL_VERSION]]/variation/indexed_vep_cache/homo_sapiens_vep_[[SPECIESDEFS::ENSEMBL_VERSION]]_GRCh38.tar.gz
-tar xzf homo_sapiens_vep_[[SPECIESDEFS::ENSEMBL_VERSION]]_GRCh38.tar.gz</pre>
-    </li>
-    <li>
-      <p><b>Non-indexed cache</b> (<a href="[[SPECIESDEFS::ENSEMBL_FTP_URL]]/release-[[SPECIESDEFS::ENSEMBL_VERSION]]/variation/vep/">[[SPECIESDEFS::ENSEMBL_FTP_URL]]/release-[[SPECIESDEFS::ENSEMBL_VERSION]]/variation/vep/</a>), e.g.:</p>
-      <pre class="code sh_sh left-margin">cd $HOME/.vep
```

Comment:
Hi @jamie-m-a , do we want to remove the bullet point and indentation since we do not have a list of items ? Current page looks like this: http://wp-np2-25.ebi.ac.uk:6080/info/docs/tools/vep/script/vep_cache.html.

Reply:
Agreed, good spot, I've removed the list tag.